gulpfile.mjs

```diff
@@ -525,11 +525,11 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + '/**/*.html', PATHS.dist + '/**/*.json'])
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
     .pipe(replace('[ios.minimum-app-version]', '1.5.3'))
-    .pipe(replace('[ios.current-app-version]', '3.0.0'))
+    .pipe(replace('[ios.current-app-version]', '3.0.1'))
     .pipe(replace('[android.latest-os-version]', '13'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
     .pipe(replace('[android.minimum-app-version]', '1.0.4'))
-    .pipe(replace('[android.current-app-version]', '3.0.1'))
+    .pipe(replace('[android.current-app-version]', '3.0.2'))
```

Comment:
There was no release information / GitHub release for that version yet. Is this planned for later today?